upgrade/1.2/Stage_1.md

```diff
@@ -71,4 +71,4 @@ Directory location = /etc/cray/ceph/_upgraded
 
 **`NOTE:`** if conman is not running please see [establishing conman console connections](operations/../../../operations/conman/Establish_a_Serial_Connection_to_NCNs.md)
 
-Once the `Stage 1` upgrade is complete please proceed to [Stage 2](Stage_2.md)
\ No newline at end of file
+Once the `Stage 1` upgrade is complete please proceed to [Stage 2](Stage_2.md)
```

Comment:
Are we moving 1.0 stage 2 to stage 1?

Reply:
great catch -- will remove current stage 1 and move others up